docs/partners/1-badging/available-badges/ed-fi-implementation-partner-badge.md

```diff
@@ -0,0 +1,66 @@
+    implementation services.
+  * The verified vendor implementation must also be badged/certified to
+        qualify as a verification. This is required to ensure that
+        implementation services are following best practices and current
+        certification/badging requirements.
+  * Collaboratives or regional service centers can provide verification if
+        their solution is badged as a “Ed-Fi Managed ODS API Platform Badge”
+  * Ed-Fi may use a developer to look at code of the implementation to look
+        for best practices (e.g. look for multi-threading in code to support
+        best practice)
+  * Verifications from individual LEAs\\SEAs require a deeper validation
+        than other means to ensure adherence to Ed-Fi standards and
+        repetability.  Vendors should consider productizing their service for
+        more streamlined scale-ability.  Part of this verification includes
+        validation Ed-Fi core components are used, code inspection to validate
+        that best practices are being used, and reviewing processes exist for
+        repeatability.
+* **Verifications - Quantity**
+    Unlike other badges, the Implementation Partner badge requires two
+    verifications. Due to the custom nature of implementations Ed-Fi requires a
+    second verification to demonstrate repeatability in the marketplace.
+* **Proof of repeatability
+    **Implementation partners must demonstrate their services are “productized”
+    for future customers. Examples of evidence includes re-usable development
+    packages, re-usable project plans, and/or detailed implementation
+    documentation. These documents can be internal intellectual property but
+    must be shared with Ed-Fi with enough detail to verify their existence and
+    usage.
+* **Participation in the Ed-Fi community
+    **Implementation partners must be in the Ed-Fi partner program. Partnership
+    requires active participation in governance and the community.
+    Implementation partners enable Ed-Fi throughout the community and provide a
+    conduit between their customer base and current Ed-Fi practices.
+* **Academy Track
+    **Implementation Partner should have at least 1 person on staff for each
+    Ed-Fi project that has gone through all the Academy tracks involving Ed-Fi
+    overview and best practices.
+* **Ed-Fi Project Retrospective Participation**Any project can be reported to
+    Ed-Fi for a group retrospective. This can be done during mid-project reviews
+    or as the project is closed out.  Lessons learned from the retrospective can
+    identify gaps in best practices and will be used to anonymously update
+    Ed-Fi’s best practices and guidance. Badge removal may occur if the
+    retrospective identifies gaps in quality and repeatability by a badged
+    vendor that are not resolved after an appropriate time.
+
+**Provisional Badges
+**The requirement for existing successful customers can be a challenging
```

Comment:
Same bold `**` problem as above. Suggestion: change this to a second level heading instead.